worlds/messenger/options.py

```diff
@@ -88,7 +88,7 @@ class ShuffleTransitions(Choice):
 
 
 class Goal(Choice):
-    """Requirement to finish the game."""
+    """Requirement to finish the game. To win power seal hunt you enter the Music Box through the shop chest."""
```

Comment:
Up to you whether you want "power seal hunt" or "Power Seal Hunt", but I think these words can help a bit
```suggestion
    """Requirement to finish the game. To win with power seal hunt, you must enter the Music Box through the shop chest."""
```